src/index.js

```diff
@@ -142,31 +143,12 @@ function loadSettings () {
  * @param   {string}    path      The absolute path to the config file
  */
 function loadConfig (path) {
-  /**
-   * @type {TSConfig}
-   */
-  const json = loadJson(path)
-  if (json) {
-    const { compilerOptions } = json
-    if (compilerOptions) {
-      const { baseUrl, paths } = compilerOptions
-
-      // Note that paths in the extending file take priority over paths in the extended file
-      // https://stackoverflow.com/questions/53804566/how-to-get-compileroptions-from-tsconfig-json
-      if (paths) {
-        settings.configFile = path
-        config.rootUrl = Path.dirname(path)
-        config.baseUrl = baseUrl || ''
-        config.paths = paths
-        return true
-      }
-    }
-
-    // if no paths, check extends
-    if (json.extends) {
-      return loadConfig(resolve(Path.dirname(path), json.extends))
-    }
-  }
+  const { compilerOptions: { baseUrl = '', paths = {} } } = parseTsconfig(path)
```

Comment:
If you wonder why I removed the `if (compilerOptions)` check: `parseTsconfig` either throws an exception if `path` does not exist, or returns an object containing `compilerOptions`. If the config file contains syntax errors that prevent `get-tsconfig` from parsing it, default `compilerOptions` are returned. Seems to be the way TS handles this too.

Reply:
Do we need to catch and represent the error as an Alias HQ issue if path does not exist, or will it be obvious from the context what it is and what needs to be done by the user?

Reply:
Cool!

I'll treat this locally later then it looks good to go